src/lib.rs

```diff
@@ -355,6 +355,7 @@ macro_rules! munge {
 
 #[cfg(test)]
 mod tests {
+    #![allow(clippy::approx_constant)]
```

Comment:
This is kind of annoying, I guess the uses of 3.14 and 1.41 could be replaced by `f32::consts::PI` and `f32::consts::SQRT_2`.

Reply:
then i fix it now